conf.ini

```diff
@@ -0,0 +1,5 @@
+[test]
```

Comment:
why is this section named test? You should probably name this section "openstack" since it involves openstack.